src/translations/translationHelpers.ts

```diff
@@ -0,0 +1,23 @@
+import { LANGS } from './constants.js';
+
+function getCurrentLanguageValue() {
+    const value = navigator.language.split(/[-_]/)[0]
+    
+    return getLangByValue(value) ? value : getDefaultLang().value
+}
```

Comment:
Please, don't use browser settings here - parse url for get an language

english urls (example):
http://the-lessoner/home
http://the-lessoner/level1/level2

russian urls (example):
http://the-lessoner/ru/home
http://the-lessoner/ru/level1/level2

Reply:
Wrote code that depends on location.pathname